tests/src/test/scala/rsc/tests/BytecodeTests.scala

```diff
@@ -14,7 +14,7 @@ class BytecodeTests extends RscTests {
       semanticFiles,
       quiet = true
     )
-    val problems = Main.process(reporter, settings)
+    val problems = MainCommand.process(reporter, settings)
```

Comment:
Imo, make `MainCommand` as private as possible and test via `Main`, as to restrict public API.